### ISSUE TITLE
Category filter button doesn't have a label so it is read out as "button" by text-to-speech

### ISSUE THREAD

**Issue by declanslevin** [redbadger/pride-london-app#120]
## Description of issue

When tapping the Category filter button with a text-to-speech tool it reads out "button" without a label to describe what the button does. 

## Expected behaviour

When tapped, the text-to-speech tool should read out "Button - Category filter" (Android) or "Category filter - Button" (iOS).

## Steps to reproduce

- Enable your device's text-to-speech function
- Go to the events page in the Pride App 
- Tap on the Category filter button
> It reads out "button" 

## OS / Browser / Device model

Galaxy S8 (Android 8.0)
iPhone 7+

**Comment by zoeabryant** [comment]
<img width="411" alt="screen shot 2018-05-11 at 14 25 15" src="https://user-images.githubusercontent.com/5781504/39926459-3736b710-5527-11e8-8188-6cf306bc15e5.png">

<img width="426" alt="screen shot 2018-05-11 at 14 25 27" src="https://user-images.githubusercontent.com/5781504/39926460-374d679e-5527-11e8-97de-3c89f2cb565d.png">

<img width="55" alt="screen shot 2018-05-11 at 14 25 41" src="https://user-images.githubusercontent.com/5781504/39926461-3762b1b2-5527-11e8-9654-06a151a4104e.png">

**Comment by zoeabryant** [comment]
I can't figure out why when you tap `Showing` it reads `Events` instead - we haven't set it as a label anywhere! 🤔

**Comment by zoeabryant** [comment]
I have looked into adding the `Filtering by x, y, z...` label - it doesn't actually read that out instead of whichever category you picked. On the web I'd usually add some hidden text to be read out instead, but it doesn't look like that concept works on apps - you need to tap to read.

**Comment by frigus02** [comment]
The word "Showing" is read as "Events" because of the `accessibilityLabel`:
https://github.com/redbadger/pride-london-app/blob/5728b7edabfef670ce59d00cdee3b07410ce41e5/src/components/FilterHeaderCategories.js#L52
I think this is wrong. I am not sure what is should read, though. Should it say "Showing" or maybe "Showing all event types"?

I agree that once a category was selected, the whole thing should read as "Filtering by x, y, z". We might be able to achieve this by giving the [corresponding ScrollView](https://github.com/redbadger/pride-london-app/blob/5728b7edabfef670ce59d00cdee3b07410ce41e5/src/components/CategoriesPills.js#L71) (or a parent) an `accessibilityLabel`. Maybe we need to hide the children from screen readers additionally by using `accessibilityElementsHidden`.

In addition to this I wonder if the headline should contain the word "Events" somehow. E.g. "Showing events of all categories" or "Showing events filtered by x, y, z".

**Comment by RGBboy** [comment]
Just to update, I have just merged #277. This is supposed to fix part of this issue. @declanslevin would you mind double checking and report back any further changes that you would like to see/hear?

**Comment by declanslevin** [comment]
I've tested on Android and iOS and the following still needs to be done: 

Tapping "Showing" should read out `Showing`
Tapping "All event types" should read out `Filter by event types`

With categories selected:

Tapping on the bar it should read out `Filtering event type by x, y, z`
